test/cql-pytest/test_cast_data.py

```diff
@@ -100,6 +100,21 @@ def test_cast_from_large_varint_to_varchar(cql, table1, cassandra_bug):
     cql.execute(f'INSERT INTO {table1} (p, cVarint) VALUES ({p}, {v})')
     assert [(str(v),)] == list(cql.execute(f"SELECT CAST(cVarint AS varchar) FROM {table1} WHERE p={p}"))
 
+# In test_cast_column_names we checked that columns are named in Cassandra style
+def test_cast_column_names(cql, table1, cassandra_bug):    
+    p = unique_key_int()
+    v = 32767456456456456456545678943512357658768763546575675
+    for t in ['int', 'double', 'text', 'tinyint', 'smallint']:
+        cql.execute(f'INSERT INTO {table1} (p, cVarint) VALUES ({p}, {v})')
+        assert cql.execute(f"SELECT CAST(cVarint as {t}) FROM {table1} WHERE p={p}").one()._fields[0] == f"cast_cvarint_as_{t}"
```

Comment:
It is sad that we need to check that version of the column name after it was botched ("cleaned") up by the Python driver (see _clean_column_names() in https://docs.datastax.com/en/drivers/python/2.7/_modules/cassandra/query.html), instead of the original name with spaces.
If you want to try something cool (and I don't insist, but please do try), you can consider telling the the driver to a different "factory", which doesn't "clean up" the column names. For example, something like (untested!):
```python
    saved_row_factory = cql.row_factory
    cql.row_factory = cassandra.query.dict_factory
    rows = cql.execute("SELECT CAST....")
    cql.row_factory = saved_row_factory
    assert rows[0].keys() == [f'cast(cvarint as {t}']
```

Reply:
Hello @nyh, I've tried this and got "ValueError: Cannot set Session.row_factory while using Configuration Profiles. Set this in a profile instead". I'll try to find a solution but not sure about the time. Actually as far as cassandra test covers this case we don't need this test at all.

Reply:
You can feel free to ignore this idea - I'll do it later (just because I think it's cool). The test we already have, with those silly underscores that the Python driver adds, is good enough already. Just please look at my other comments, and I'll be happy to merge the next version.

Reply:
@nyh , Done